edx_sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -137,12 +139,13 @@ def translate_course_run(self, course_run, course, program_dictionary=None):
             sailthru_content['images'] = {'thumb': {'url': course_run['image']['src']}}
 
         # create the interest tags
-        tags = self._get_tags_from_property(course.get('subjects'), 'name', 'subject')
+        tags = ['course-run']
+        tags.extend(self._get_tags_from_property(course.get('subjects'), 'name', 'subject'))
         tags.extend(self._get_tags_from_property(course.get('owners'), 'key', 'school'))
         tags.extend(self._get_tags_from_property(course.get('sponsors'), 'key', 'school'))
 
         if len(tags) > 0:
```

Comment:
Nit: there is no longer a way for this condition to be `false` with the new code.